.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Report a bug
+about: Found something? Thank you for telling us!
+labels: ["bug"]
+---
+
+## System details
+
+    Publisher Version        :
+    Connect Version          :
+    Your OS                  :
+    Content Type to Deploy   :
+
+## Publisher Output and Logs
+
+<!--
+Publisher output helps us to identify the problem,
+it is located in the "Output" window under the "Posit Publisher Deployment" option.
+See https://github.com/posit-dev/publisher/blob/main/docs/troubleshooting.md#publisher-output
```

Comment:
This is how the new section in troubleshooting will look like

---

<img width="1271" alt="Screenshot 2025-05-14 at 12 06 10 p m" src="https://github.com/user-attachments/assets/7e49ec1f-2c94-4cc9-b7a7-762836cdc4e7" />